includes/admin/class-evf-admin-settings.php

```diff
@@ -276,7 +279,7 @@ public static function output_fields( $options ) {
 					case 'tel':
 						$option_value = self::get_option( $value['id'], $value['default'] );
 
-						?><tr valign="top">
+						?><tr valign="top" class="<?php echo esc_attr( $value['parent_class'] ); ?>">
```

Comment:
Is there any situation we have to introduce `parent_class` for all. I think let's introduce it for the radio field and leave others as it is!

Reply:
@Nilanzva any reason over this?

Reply:
I have to introduce the parent class because while loading the reCaptcha setting page all of the input was visible ( V2 and V3 site and secret key field ) and after few secs only the appropriate input was visible.

Reply:
I have introduced `is_visible` args acceptable, please see changes :)